test/ext/asyncio/test_engine_py3k.py

```diff
@@ -590,6 +591,15 @@ async def test_create_async_engine_server_side_cursor(self, async_engine):
             server_side_cursors=True,
         )
 
+    def test_async_engine_from_config(self):
+        config = {
+            "sqlalchemy.url": str(testing.db.url),
+            "sqlalchemy.echo": "true",
+        }
+        engine = engine_from_config(config, create_engine=create_async_engine)
+        assert engine.url == testing.db.url
+        assert engine.echo is True
```

Comment:
We should also check that engine.is_async is true (or _is_async don't remember atm)

Reply:
I guess it's `engine.dialect.is_async`, right?